### ISSUE TITLE
Accessibility manifesto

### ISSUE THREAD

**Issue by sarahesbie** [codelikeagirlau/accessibility-resources#20]
I love the idea of our site having a manifesto that collects together all of the values and ideas we're sharing around accessibility. Things like, who in an organisation should be responsible for accessibilty, etc. I also love the idea of this manifesto being open source. I'm keen to get the ball rolling on getting this set up, and then see what everyone else wants to contribute.
